src/dynamo0p3.py

```diff
@@ -1569,17 +1794,28 @@ def gen_code(self, parent):
                 TypeDeclGen(invoke_sub,
                             datatype="operator_proxy_type",
                             entity_decls=op_proxy_decs))
+        cma_op_proxy_decs = self.unique_proxy_declarations(
+            "gh_columnwise_operator")
+        if len(cma_op_proxy_decs) > 0:
+            invoke_sub.add(
+                TypeDeclGen(invoke_sub,
+                            datatype="columnwise_operator_proxy_type",
+                            entity_decls=cma_op_proxy_decs))
+
         # Initialise the number of layers
         invoke_sub.add(CommentGen(invoke_sub, ""))
         invoke_sub.add(CommentGen(invoke_sub, " Initialise number of layers"))
         invoke_sub.add(CommentGen(invoke_sub, ""))
 
         # Use the first argument that is not a scalar
```

Comment:
Oops, don't know what I did there. I've added a comment in the relevant location.